lib/engines/content_block_manager/app/components/content_block_manager/content_block_edition/details/fields/call_charges_component.rb

```diff
@@ -10,4 +10,8 @@ def call_charges_info_url
   def label
     field.nested_field("label")
   end
+
+  def label_for(name)
```

Comment:
There's actually a helper method here you can use - https://github.com/alphagov/whitehall/blob/8803ac6006763f7d08490f1b25ad958e3fa33442/lib/engines/content_block_manager/app/helpers/content_block_manager/content_block/translation_helper.rb#L2

Reply:
👍 thanks, will tweak

Reply:
@pezholio this is now tweaked to use the existing helper, so ready for another look.